org.lflang/src/org/lflang/validation/AttributeSpec.java

```diff
@@ -210,9 +239,26 @@ enum AttrParamType {
         ));
         // @sparse
         ATTRIBUTE_SPECS_BY_NAME.put("sparse", new AttributeSpec(null));
+        // @language(lang)
+        ATTRIBUTE_SPECS_BY_NAME.put("language", new AttributeSpec(
+            List.of(new AttrParamSpec(AttributeSpec.VALUE_ATTR, AttrParamType.LANGUAGE, null))
+        ));
+        // @_fed_recv
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_recv", new AttributeSpec(null));
+        // @_fed_send
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_send", new AttributeSpec(null));
+        // @_fed_inp_ctrl
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_inp_ctrl", new AttributeSpec(null));
+        // @_fed_out_ctrl
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_out_ctrl", new AttributeSpec(null));
+        // @_fed_config
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_config", new AttributeSpec(
```

Comment:
The only use of this attribute appears to be to tell the TS generator that the main reactor is a federate. Why don't we simply use a `@federate` attribute without argument for this?

Reply:
Fixed in https://github.com/lf-lang/lingua-franca/pull/1552